deploy/src/main/deploy/kubernetes/olm_deploy.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2018 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+set -e
+
+SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
+source "$SCRIPTPATH/common.sh"
+
+echo DEPLOYING OPERATOR LIFECYCLE MANAGER TO KUBERNETES
+
+if [ $(is_olm_deployed) == "YES" ]; then
+	echo "OLM already deployed. Exiting ..."
+	exit 0
+fi
+
+
+echo
+echo "Deploying OLM ..."
+test -d operator-lifecycle-manager || git clone https://github.com/operator-framework/operator-lifecycle-manager.git
+kubectl create -f operator-lifecycle-manager/deploy/upstream/manifests/latest/ || true
```

Comment:
I have some concerns here using `latest` since it might result in undeterministic behavior due to changes in the Git repository. I also wonder if we cannot replace the whole cloning of the repo with some explicit URLs to the resources in the 0.8.0 folder.

Reply:
I agree, so I pinned the checkout to the specific commit that I tested. I am not sure though how the git clone operation could be replaced.

Reply:
well, we could simply do something like
```
kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/manifests/0.8.0/0000_30_00-namespace.yaml
kubectl create -f https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/manifests/0.8.0/0000_30_01-olm-operator.serviceaccount.yaml
...
```
since it is only 18 resources, it shouldn't be too bad to deploy them individually. This might also help with the timing problem that requires us to re-deploy some of the resources ...

Reply:
Done